src/features/events/EventCard.tsx

```diff
@@ -6,15 +6,17 @@ export type CFTEvent = {
 }
 
 const EventCard = ({ event }: { event: CFTEvent }) => {
+	const title: string = event.title.toUpperCase();
+
 	return (
-		<article className='bg-cft-light-gray p-6 text-cft-black'>
-			<h1>{event.title}</h1>
-			<h2>{event.date}</h2>
-			<h2>{event.location}</h2>
+		<article className='bg-[#133A4E] text-white p-4 shadow-lg shadow-red-900'>
```

Comment:
add the dark:prefix to colors and add the light theme colors as default values. 
also looks like  #133A43 is set in tailwind.config.js as `cft-blue`. Looking at the actual color values, lets rename it to '`cft-bluegreen` in both places and lose the bracket notation.